webapp/src/components/custom_post_types/post_type/component.tsx

```diff
@@ -136,6 +138,8 @@ const PostType = ({
         );
     }
 
+    const compactTitle = compactDisplay ? '\u200b' : '';
```

Comment:
Any reason for not using something more semantically defined like the `<br>` tag?